drivers/ioexpander/gpio.c

```diff
@@ -569,7 +612,7 @@ int gpio_pin_register(FAR struct gpio_dev_s *dev, int minor)
       case GPIO_OUTPUT_PIN_OPENDRAIN:
         {
           DEBUGASSERT(dev->gp_ops->go_read != NULL &&
-                      dev->gp_ops->go_write != NULL);
+                     dev->gp_ops->go_write != NULL);
```

Comment:
the old code is right?

Reply:
@xiaoxiang781216 - thanks -I restored the white space